### PR TITLE
Add resource remove feature

### PR DESCRIPTION
**Pull request by okucheruk**
This PR add resource remove feature.

`Highlights`
- `RemoveAction` executes removing and dispatches `GetFilesAction`.
- `Notyf` toasts added to improve UX.

### CODE REVIEW

**Comment by okucheruk** [comment]
@dkalinin, PTAL.

File: app/page/filebrowser/mutation/RemoveMutation.js [hunk 1]
```diff
@@ -20,11 +20,23 @@ import Mutation from '../../../lib/turbine/service/store/Mutation.js';
  * @author Oleksii Kucheruk
  */
 export default class RemoveMutation extends Mutation {
+  /**
+   * Represents RemoveMutation.
+   *
+   * @param {Object} removeModel The resource remove model object.
+   * @constructor
+   */
+  constructor(removeModel) {
```
Comment (dkalinin) [inline app/page/filebrowser/mutation/RemoveMutation.js]: modelToRemove

File: app/page/filebrowser/mutation/RemoveMutation.js [hunk 2]
```diff
@@ -20,11 +20,23 @@ import Mutation from '../../../lib/turbine/service/store/Mutation.js';
...
+    this._removeModel = removeModel;
+  }
+
   /**
    * Mutates removeFile property.
    *
    * @param {StateProxy} appState The application state object.
    */
   accept(appState) {
+    appState.changeState('lastRemove', this._removeModel);
```
Comment (dkalinin) [inline app/page/filebrowser/mutation/RemoveMutation.js]: lastRemovedModel

File: index.html [hunk 3]
```diff
@@ -8,11 +8,15 @@
...
   <link rel="stylesheet" type="text/css" href="node_modules/notyf/notyf.min.css">
+<!--  <link type="text/css" rel="stylesheet"-->
+<!--        href="node_modules/materialize-css/dist/css/materialize.min.css" media="screen,projection"/>-->
+
 </head>
 
 <body>
   <div id="index"></div>
   <script src="node_modules/notyf/notyf.min.js" type="text/javascript"></script>
+<!--  <script src="node_modules/materialize-css/dist/js/materialize.js" type="text/javascript"></script>-->
```
Comment (dkalinin) [inline index.html]: Please remove.

File: package.json [hunk 4]
```diff
@@ -23,6 +23,8 @@
     "eslint-config-google": "^0.13.0"
   },
   "dependencies": {
+    "materialize-css": "^1.0.0",
```
Comment (dkalinin) [inline package.json]: Please remove.

File: app/page/filebrowser/mutation/RemoveMutation.js (hunk 1 above)
Comment (okucheruk) [reply app/page/filebrowser/mutation/RemoveMutation.js]: Renamed.

File: app/page/filebrowser/mutation/RemoveMutation.js (hunk 2 above)
Comment (okucheruk) [reply app/page/filebrowser/mutation/RemoveMutation.js]: Renamed

File: index.html (hunk 3 above)
Comment (okucheruk) [reply index.html]: Removed.

File: package.json (hunk 4 above)
Comment (okucheruk) [reply package.json]: Removed.

**Comment by okucheruk** [comment]
@dkalinin, fixed, PTAL.

**Pull request merged**